### ISSUE TITLE
IDS not resilient against network glitches

### ISSUE THREAD

**Issue by fisherab** [icatproject/ids.server#16]
```
Tom reported that he had seen cases where the IDS had not recovered properly after network
problems.
```

Original issue reported on code.google.com by `dr.s.m.fisher` on 2014-12-19 09:32:36

**Comment by fisherab** [comment]
More information is needed to address this

**Comment by tomgriffinstfc** [comment]
ISIS are no longer seeing this problem.
We believe it was resolved in b0abb796a659973538824f08eac63ff5b3c7b4ea

I think the issue can be closed if no one else is seeing problems

**Comment by fisherab** [comment]
In that case  I will close it. It can always be re-opened
